build_js.sh

```diff
@@ -4,10 +4,12 @@
 set -e
 
 BULLET=false
+WEB_APPS=true
 
 while [[ "$#" -gt 0 ]]; do
     case $1 in
         --bullet) BULLET=true ;;
+        --no-web-apps) WEB_APPS=false ;;
```

Comment:
Seems like an odd request as their are other ways to speed up compilation (like not compiling with Release or disabling LTO). It would be nice to have tests for this, but I don't know if any of them would be useful besides does it compile with current infra.